policy_packs/aws/vpc/enforce_default_security_groups_to_not_allow_any_access/README.md

```diff
@@ -3,7 +3,7 @@ categories: ["networking", "security"]
 primary_category: "security"
 ---
 
-# Enforce AWS VPC Default Security Groups to Not Allow Any Access
+# Enforce AWS VPC Default Security Groups to Deny All Access
```

Comment:
I'm not sure what this policy pack is doing, is it only looking at default VPC security groups and blocking ingress/egress access?

Reply:
Yes, all Ingress and Egress rules are reject for default VPC security groups.